requirements.txt

```diff
@@ -3,4 +3,5 @@
 nose
 rednose
 coverage
+smart-sentinel>=0.2.0
```

Comment:
It's failing because we need to run redis-sentinel for tests on Travis and Travis, by default has only Redis service.
I'm working to solve it.